tests/fingerprints-icons/.gitignore

```diff
@@ -0,0 +1,2 @@
+.DS_Store
```

Comment:
Please use the top-level gitignore.

Reply:
I copied the test from the existing basic test folder.

Should I remove them from all the test folders or just these tests?

Reply:
I moved gitignore settings to the top-level gitignore